front/pages/api/v1/w/[wId]/data_sources/[name]/tables/[tId]/index.ts

```diff
@@ -37,7 +38,10 @@ async function handler(
     });
   }
 
-  if (!owner.flags.includes("structured_data")) {
+  if (
+    !owner.flags.includes("structured_data") ||
+    !owner.flags.includes("auto_pre_ingest_all_databases")
```

Comment:
Not sure we want this right ? this one should just condition whether we ingest all DBs automatically but we should still allow deleting regardless no ?